client/src/pages/reports/Compact.js

```diff
@@ -0,0 +1,939 @@
+    .banner {
+      display: inline-block !important;
+      -webkit-print-color-adjust: exact;
+      color-adjust: exact !important;
+    }
+    .workflow-action .btn {
+      display: inline-block !important;
+    }
+  }
+`
+
+const CompactReportViewHeader = ({
+  onPrintClick,
+  returnToDefaultPage,
+  noReport,
+  optionalFields,
+  setOptionalFields
+}) => {
+  return (
+    <Header>
+      <HeaderTitle value="title">
+        {Settings.fields.report.compactView}
+      </HeaderTitle>
+      <SimpleMultiCheckboxDropdown
+        label="Optional Fields ⇓"
+        options={optionalFields}
+        setOptions={setOptionalFields}
+      />
+      <Buttons>
+        {!noReport && (
+          <Button
+            value="print"
+            type="button"
+            bsStyle="primary"
+            onClick={onPrintClick}
+          >
+            Print
+          </Button>
+        )}
+        <Button
+          value="detailedView"
+          type="button"
+          bsStyle="primary"
+          onClick={returnToDefaultPage}
+        >
+          {Settings.fields.report.detailedView}
+        </Button>
+      </Buttons>
+    </Header>
+  )
+}
+
+CompactReportViewHeader.propTypes = {
+  onPrintClick: PropTypes.func,
+  returnToDefaultPage: PropTypes.func,
+  noReport: PropTypes.bool,
+  optionalFields: PropTypes.objectOf(
+    PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      active: PropTypes.bool.isRequired
+    })
+  ).isRequired,
+  setOptionalFields: PropTypes.func
+}
+
+CompactReportViewHeader.defaultProps = {
+  noReport: false
+}
+
+const Header = styled.header`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  max-width: 21cm;
+`
+
+const HeaderTitle = styled.h3`
+  margin: 0;
+  @media print {
+    display: none;
+  }
+`
+
+const Buttons = styled.div`
+  margin-bottom: 1rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  button {
+    margin-left: 5px;
+    margin-right: 5px;
+  }
+`
+
+const CompactReportHeaderContent = ({ report }) => {
+  const location = useLocation()
+  const { appSettings } = useContext(AppContext)
+  return (
+    <HeaderContentS bgc={appSettings[SETTING_KEY_COLOR]}>
+      <img src={anetLogo} alt="logo" width="50" height="12" />
+      <ClassificationBanner />
+      <span style={{ fontSize: "12px" }}>
+        <Link to={location.pathname}>{report.uuid}</Link>
+      </span>
+    </HeaderContentS>
+  )
+}
+
+CompactReportHeaderContent.propTypes = {
+  report: PropTypes.object
+}
+
+const CompactReportFooterContent = () => {
+  const { currentUser, appSettings } = useContext(AppContext)
+  return (
+    <FooterContentS bgc={appSettings[SETTING_KEY_COLOR]}>
+      <img src={anetLogo} alt="logo" width="50" height="12" />
+      <ClassificationBanner />
+      <PrintedByBoxS>
+        <div>
+          printed by <LinkTo modelType="Person" model={currentUser} />
+        </div>
+        <div>{moment.utc().toString()}</div>
```

Comment:
Please format the timestamp according to the format from the settings (e.g. `Settings.dateFormats.forms.displayLong.withTime`).